Define printWelcome function.

Tauheed-Elahee opened the pull request.

Tauheed-Elahee committed:
Define printWelcome function.

Tauheed-Elahee committed:
Update printScreen.h

Added the "Bluetooth Mode" screen.

Tauheed-Elahee committed:
Update printScreen.h

lots of changes.

seththompsonn reviewed (approved):
This all looks Coolio to me my guy

rshii reviewed (changes requested):
Missing LCD declaration, make functionality to pass in LCD instance to module

Don't know if you need 'this....'

Tauheed-Elahee committed:
Make the header file have the proper form.

Tauheed-Elahee committed:
Replaced "" with " "

Tauheed-Elahee committed:
Reduced number of print and setCursor statments.

Tauheed-Elahee committed:
Change to into to char.

Tauheed-Elahee committed:
Update printScreen.h

Tauheed-Elahee committed:
Merge pull request #4 from Tauheed-Elahee/master

rebase

Commit:
Fix LCD.h then make Motor.h and pinLayout.h

Tauheed-Elahee committed:
TODO: Seperate mixing and pouring drink functions

Made comment about separating mixing and pouring drinks.

Tauheed-Elahee committed:
Updated SD card pins

Updated SD card pins so that the layout would be consistent with everything else.

Tauheed-Elahee committed:
Update LCD.h

Look at how to declare a string!!!

Tauheed-Elahee committed:
Update pinLayout.h

Added the setup() function.

Tauheed-Elahee committed:
Update pinLayout.h

Call the pinLayout and set it up in the LCD constructor.

Tauheed-Elahee committed:
Update LCD.h

Tauheed-Elahee committed:
Update Arduino-Pin-Layout.csv

Added the enable pin for Plinko.

Tauheed-Elahee committed:
Update pinLayout.h

#define stuff

Tauheed-Elahee committed:
Update pinLayout.h

Tauheed-Elahee committed:
Update Drinking-Machine-Arduino-Code.ino

Test

Tauheed-Elahee committed:
Update Drinking-Machine-Arduino-Code.ino

Tauheed-Elahee committed:
Update Drinking-Machine-Arduino-Code.ino

Tauheed-Elahee committed:
Update pinLayout.h

Re-did pinLayout.h

Tauheed-Elahee committed:
Update LCD.h

Update variable names.

Tauheed-Elahee committed:
Merge pull request #6 from Tauheed-Elahee/master

Rebase

Tauheed-Elahee committed:
Merge pull request #7 from Tauheed-Elahee/master

Used Kdevelop's advanced analysis tools to fix scoping issue.

Commit:
stuff

Commit:
why git

Commit:
Used Kdevelop

Tauheed-Elahee committed:
Delete Drinking-Machine-Arduino-Code.kdev4

Tauheed-Elahee committed:
Delete Drinking-Machine-Arduino-Code.kdev4

Commit:
Testing with Kdevelop.

Commit:
Removed comment in PinLayout.h

Commit:
Testing with Kdevelop.

Commit:
Deleted Old Libraries

Commit:
still need to commit added files through terminal

Commit:
still need to commit added files through terminal

Commit:
Fixed String class and LCD() constructor.

Commit:
Remove unneccesary link to WCharacter.h.

Commit:
still need to commit added files through terminal

Commit:
still need to commit added files through terminal

Commit:
Added files to test LCD on Arduino

Tauheed-Elahee committed:
rebase (#8)

* comment

* Delete Drinking-Machine-Arduino-Code.kdev4

* Delete Drinking-Machine-Arduino-Code.kdev4

* Update README.md (#5)

Tauheed-Elahee committed:
rebase (#9)

* comment

* Delete Drinking-Machine-Arduino-Code.kdev4

* Delete Drinking-Machine-Arduino-Code.kdev4

* Update README.md (#5)

Tauheed-Elahee committed:
Delete LCD test files including libraries.zip

Remove file.

Tauheed-Elahee committed:
Delete Drinking-Machine-Arduino-Code-Makefile.mk

Remove make files.

Tauheed-Elahee committed:
Delete Drinking-Machine-Arduino-Code.kdev4

Remove Kdevelop files.

Tauheed-Elahee committed:
Delete Drinking-Machine-Arduino-Code.kdev4

Remove KDevelop files.

Tauheed-Elahee committed:
Delete Motor.h

Remove duplicated effort of motor library.

Tauheed-Elahee committed:
Change file paths to fit arduino IDE

Tauheed-Elahee committed:
Change file paths for Arduino IDE

Tauheed-Elahee committed:
Change file paths for Arduino IDE

Tauheed-Elahee committed:
Change file paths for Arduino IDE

Tauheed-Elahee committed:
Change pin layout from 11 and 12 to 6 and 7

Tauheed-Elahee committed:
Remove unnecessary link to WCharacter library

Tauheed-Elahee committed:
Change pins 11, 12, 13 to 6, 7, 10

Tauheed-Elahee committed:
Change speakerOutput from pin 13 to pin 10.

Tauheed-Elahee committed:
Remove builtin Arduino libraries to transistion development environment to Arduino IDE

Tauheed-Elahee committed:
remove makefiles

Tauheed-Elahee committed:
Added the Buttons library to handle the buttons

Tauheed-Elahee committed:
Added the Buttons library to handle the buttons

Tauheed-Elahee committed:
Changed the game wining screen
